hermes-common/src/test/groovy/pl/allegro/tech/hermes/common/metric/counter/zookeeper/CounterMatcherTest.groovy

```diff
@@ -34,6 +34,22 @@ class CounterMatcherTest extends Specification {
         subscriptionName == "subscription"
     }
 
+    def "should match subscription delivered with underscores in name"() {
+        given:
+            def counterName = "delivered.pl_allegro_offercore.topic.auction_offer-ended-only"
```

Comment:
can we stick to the given/when/then code style without such indentations, just to be consistent (even in this single file)?

Reply:
Sorry, my bad

Reply:
I mean, it's fine, some people will say it's more readable (I personally prefer the no-indentation way as it's presented in spock documentation's examples http://spockframework.org/spock/docs/1.0/spock_primer.html), it simply would be nice to have only a single style across our code. If we change our minds, let's change it globally. :)

Reply:
Ah, ok, got it. Yeah, in the team we can discuss preferable style and change it gradually, but currently it will be easier to unindent my test case.

Reply:
Great, thanks!